client/src/components/Group.js

```diff
@@ -0,0 +1,73 @@
+import styled from "styled-components";
+import { bool } from "prop-types";
+import { NavLink } from "react-router-dom";
+
+import IconGroup from "./IconGroup";
+import Button from "./Button";
+import IconUser from "./IconUser";
+
+export default function Group({ openGroup, setOpenGroup }) {
+  return (
+    <NavContainer openGroup={openGroup}>
+      <IconGroup />
+      <h4> Groupname </h4>
+      <br />
+      <p> Berlin</p>
+      <p>
+        Apples ducks straw, quail a ostriches donkey, hay hook cucumbers. Lamb
+        pig rooster sheep. . Onion organic orange
+      </p>
+      <Flexbox>
+        <IconUser />
+        <IconUser />
+        <IconUser />
+        <IconUser />
+        <IconUser />
+      </Flexbox>
+
+      <Button>
+        <Link to="./editGroup" onClick={() => setOpenGroup(!openGroup)}>
```

Comment:
Interesting, does that work with the `.` before the slash? 🤔

Reply:
it does. but I changed it and it still works.